Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -274,6 +275,13 @@ - (SentrySpanStatus)spanStatusForHttpResponseStatusCode:(NSInteger)statusCode
     return kSentrySpanStatusUndefined;
 }
 
+- (NSString *)removeSentryTraceIdFromBaggage:(NSString *)baggage
+{
+    NSMutableDictionary *original = [SentrySerialization decodeBaggage:baggage].mutableCopy;
+    [original removeObjectForKey:@"sentry-trace_id"];
+    return [SentrySerialization baggageEncodedDictionary:original];
```

Comment:
We should remove every sentry item from the baggage, not only the trace_id.

Reply:
I wasn't sure if that was necessary and in the end I thought not. After all we're sending this to Sentry, and apart from the trace id it all seems safe to send?

Can you explain why we should remove all of them?

Reply:
Just to play nice with the app. We shouldn't send any byte that is not necessary.

Reply:
Done